apps/wakatime/main.py

```
@@ -62,106 +67,122 @@ def authorize():
     data = {
         "client_id": client_id,
         "client_secret": client_secret,
         "grant_type": "authorization_code",
         "code": code,
-        "redirect_uri": redirect_uri
+        "redirect_uri": redirect_uri,
     }
-    
+
     try:
         response = requests.post(token_url, data=data)
```

Ruff (0.14.14)

Probable use of `requests` call without timeout

(S113)